DP02_Galaxy_Cluster_Weak_Lensing.ipynb

```diff
@@ -0,0 +1,769 @@
+{
```

Comment:
<ul><li>Consider adding a more ellaborated desciption to each one of these, so that users have a better idea on how to implement them </li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rubin-dp0/tutorial-notebooks/pull/228/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>

Reply:
<p>adjusted</p>